addon/chrome.manifest

```diff
@@ -1 +1,2 @@
 content screenshots-skin chrome/skin/
+content screenshots chrome/content/
```

Comment:
@Standard8 : do you know what the right jar.mn incantation is to include `addon/chrome/content/localizedTooltips.jsm` in the build/tree/jar?